client/my-sites/plugins-wpcom/plugin.jsx

```diff
@@ -42,10 +54,12 @@ export const Plugin = React.createClass( {
 			: icon;
 
 		return (
-			<div className="wpcom-plugins__plugin-item">
+			<div
+				className="wpcom-plugins__plugin-item"
+			>
 				<a
 					href={ descriptionLink }
-					onClick={ onClick }
+					onClick={ () => onClick( name ) }
```

Comment:
This violates [jsx-no-bind](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md).
This doesn't seem to need a binding, and a normal function would not violate the rule.

Everything else LGTM.